src/utils/certificates_generator.py

```diff
@@ -8,10 +8,20 @@
 from src.utils.api_helper import ApiHelper
 from src.utils.db_postgress_helper import DbConnect
 
+fake = Faker('en_CA')
+C_NAMES = [fake.company() for x in range(0, 5)]
+
+
+def gen_date():
+    expiration_date = "{}-{}-{}T15:15:10.440Z".format(
+        random.randint(2017, 2023),
+        random.choice(["%.2d" % i for i in range(1, 12)]),
+        random.choice(["%.2d" % i for i in range(1, 25)]))
+    return expiration_date
```

Comment:
no need to implement this
just get current time stamp ( datetime.datetime.now() ), extend it and format it

Reply:
and use inside generate_payload()